src/mbgl/geometry/anchor.hpp

```diff
@@ -6,17 +6,15 @@
 
 namespace mbgl {
 
-struct Anchor {
+class Anchor {
+public:
     Point<float> point;
     float angle = 0.0f;
     float scale = 0.0f;
     int segment = -1;
 
-    explicit Anchor(float x_, float y_, float angle_, float scale_)
```

Comment:
Here, no purpose. `explicit` is used to prevent [unintended implicit conversions via a single-argument constructor](http://stackoverflow.com/questions/121162/what-does-the-explicit-keyword-in-c-mean). This isn't a single-argument constructor, so that doesn't apply.

Using `explicit` on a multi-argument constructor does prohibit the type name `Anchor` from being omitted with initializer lists such as:

```
Anchor getAnchor() {
    return { x, y, angle, scale };
}
```

But I'm not aware of any cases where we actually want that behavior.